bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/impl/ZeebeConstants.java

```diff
@@ -64,4 +68,6 @@
   public static final String USER_TASK_FORM_KEY_CAMUNDA_FORMS_FORMAT = "camunda-forms";
   /** Form key location used for forms embedded in the same BPMN file, i.e. zeebeUserTaskForm */
   public static final String USER_TASK_FORM_KEY_BPMN_LOCATION = "bpmn";
+
+  public static final String ELEMENT_CALLED_DECISION = "calledDecision";
```

Comment:
🔧  Please move this closer to the `ELEMENT_*` properties, so these are all kept together.